packages/modal/src/Modal.js

```diff
@@ -208,7 +209,7 @@ export default class Modal extends React.Component<Props, State> {
                   <h6>{title}</h6>
                 )}
                 {hasCloseButton && (
-                  <Button className="m-modal__close" icon="times" type="default" transparent onClick={() => this.handleRequestClose()} />
+                  <Button className="m-modal__close a-button-transparent" icon="times" type="default" onClick={() => this.handleRequestClose()} />
```

Comment:
Zou je niet verwachten dat als je het attr transparent op de Button zet, de Button zelf ervoor zorgt dat de juiste klasse wordt toegevoegd?

Reply:
Dit is een hyaat in de core branding die er sinds het begin inzit. Je hebt `...-transparent`, maar ook `...--transparent`. Het attribuut `transparent` zorgt voor het tweede, terwijl ik het eerste nodig heb.

Reply:
normaal zou dit idd zo moeten werken

```
    const btnClass = classNames(
      className,
      {
        'a-button': !outline && !negative,
        [`${typeClasses[type]}`]: !!type,
        [`${sizeClasses[size]}`]: !!size,
        'has-icon': !!icon,
        'has-icon-left': !!iconLeft,
        'has-icon-right': !!iconRight,
        'a-button-negative': !!negative,
        'a-button-outline': !!outline,
        'a-button-transparent': !!transparent,
        'a-button--block': block,
      }
    );
```

Reply:
Thumbs down verklaring: zie [comment hierboven](https://github.com/digipolisantwerp/acpaas-ui_react/pull/144#discussion_r415774466).